app/controllers/documents_controller.rb

```diff
@@ -0,0 +1,69 @@
+class DocumentsController < ApplicationController
+  def index
+      @documents = Document.all.order(created_at: :desc)
+  end
+
+  def new
+    @document = Document.new
+  end
+
+  def create
+    @document = Document.new
+    @document.title = params[:document][:title]
+    @document.description = params[:document][:description]
+    @document.keywords = params[:document][:keywords]
+    @document.user_id = current_user.id
```

Comment:
I think this is part magic.  I reduced 5 lines to this:
if current_user.documents.build(document_params).save

Can we talk about this?